tests/proxysql-admin-testsuite.bats

```diff
@@ -121,13 +121,24 @@ fi
 }
 
 
+@test "run proxysql-admin --update-mysql-version ($WSREP_CLUSTER_NAME)" {
+  run sudo PATH=$WORKDIR:$PATH $WORKDIR/proxysql-admin --update-mysql-version
+  echo "$output" >&2
+  [ "$status" -eq  0 ]
+}
+
+@test "run the check for --update-mysql-version ($WSREP_CLUSTER_NAME)" {
+  local check_mysql_version=$(mysql_exec "$HOST_IP" "$PORT_3" "SELECT VERSION();" | tail -1 | cut -d'-' -f1)
+  local check_proxysql_mysql_version=$(proxysql_exec "select variable_value from global_variables where variable_name like 'mysql-server_version'" | awk '{print $0}')
+  [ "$check_proxysql_mysql_version" == "$check_mysql_version" ]
```

Comment:
It's useful to output the version values before the test, so if it fails we can see what the values were (see the --version check in generic-test.bats)

Reply:
Done